perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -1836,9 +1837,14 @@ private String substituteCommonStrings(Application app, List<ApplicationFormItem
 			String footer = "";
 			// get proper value from attribute
 			try {
-				Attribute attrFooter = attrManager.getAttribute(registrarSession, app.getVo(), URN_VO_MAIL_FOOTER);
-				if (attrFooter != null && attrFooter.getValue() != null) {
-					footer = BeansUtils.attributeValueToString(attrFooter);
+				Attribute attrGroupFooter = attrManager.getAttribute(registrarSession, app.getGroup(), URN_GROUP_MAIL_FOOTER);
```

Comment:
Hi, before getting group attribute you should check if application has group in itself ``app.getGroup() != null``. Right now it would fail for vo notifications, since it would call getAttribute() with null object.
IF group -> fill group footer -> if empty, fill vo footer
IF vo -> fill vo footer